server/files/configure_misp.sh

```diff
@@ -101,37 +103,38 @@ apply_updates() {
 init_user() {
     # Create the main user if it is not there already
     sudo -u www-data /var/www/MISP/app/Console/cake userInit -q 2>&1 > /dev/null
-    echo "... setting admin email to '${ADMIN_EMAIL}'"
-    sudo -u www-data /var/www/MISP/app/Console/cake Admin setSetting -q "MISP.email" ${ADMIN_EMAIL}
+
     echo "UPDATE misp.users SET email = \"${ADMIN_EMAIL}\" WHERE id = 1;" | ${MYSQLCMD}
-    if [ ! -z "$ADMIN_ORG" ]; then
+
+    if [ -n "$ADMIN_ORG" ]; then
```

Comment:
Got burnt a couple of times with this since empty or undefined values are treated differently. Can you revert this?